src/components/apps/launch/AppLaunchForm.js

```diff
@@ -427,6 +437,16 @@ const AppLaunchForm = (props) => {
             )
         );
 
+    const stepperRef = React.useRef(null);
+    const [stepperHeight, setStepperRef] = useComponentHeight();
+    const appBarHeight =
+        stepperHeight +
+        (isMobile ? MOBILE_NAVIGATION_BAR_HEIGHT : NAVIGATION_BAR_HEIGHT);
+
```

Comment:
Was able to save some space for `mobile` view by removing text and reducing `margins` and `padding`.